WordPress/Classes/ViewRelated/Reader/ReaderSaveForLater+Analytics.swift

```diff
@@ -71,14 +76,20 @@ extension ReaderSaveForLaterAction {
 }
 
 extension ReaderStreamViewController {
-    func trackSavedPostNavigation() {
+    func trackSavedPostNavigation(post: ReaderPost) {
+        let blogID = post.siteID ?? 0
+        let feedID = post.feedID ?? 0
+        var properties: [String: Any] = ["blog_id": blogID,
+                                         "feed_id": feedID,
+                                         "foolow": post.isFollowing]
```

Comment:
Typo 👀 
```suggestion
                                         "follow": post.isFollowing]
```

Reply:
Oops 😬 I will definitely go for a enum key in a separate PR, but thanks for catching this anyway!